### ISSUE TITLE
Typo in javax.mail dependency

### ISSUE THREAD

**Issue by akreuzer** [bmc/avsl#5]
There is an extra dot in the dependency:
I guess it should read `com.sun.mail`

https://github.com/bmc/avsl/blob/58ac96cdb07cc80796808faebdd3b50f357c3e75/build.sbt#L30

Using version 1.0.16 I get the following error in sbt.
```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.sun..mail#javax.mail;1.6.0: java.text.ParseException: inconsistent module descriptor file found in 'https://repo1.maven.org/maven2/com/sun//mail/javax.mail/1.6.0/javax.mail-1.6.0.pom': bad organisation: expected='com.sun..mail' found='com.sun.mail';
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn]
[warn] 	Note: Unresolved dependencies path:
[warn] 		com.sun..mail:javax.mail:1.6.0
[warn] 		  +- org.clapper:avsl_2.12:1.0.16 (/Users/alex/git/glass/rules/build.sbt#L28-65)
[warn] 		  +- com.cognotekt:glass-rules_2.12:0.0.3
[error] sbt.librarymanagement.ResolveException: unresolved dependency: com.sun..mail#javax.mail;1.6.0: java.text.ParseException: inconsistent module descriptor file found in 'https://repo1.maven.org/maven2/com/sun//mail/javax.mail/1.6.0/javax.mail-1.6.0.pom': bad organisation: expected='com.sun..mail' found='com.sun.mail';
```
I guess that this is a result of the above mentioned typo.

**Comment by bmc** [comment]
Gack. My bad. I'll fix it now. I have to publish another package today anyway.

**Comment by bmc** [comment]
Fixed in version 1.0.17 and pushed to Bintray. I've requested a sync with Maven Central, but that'll likely take an hour or so.

**Comment by bmc** [comment]
Thanks for catching it and reporting it.
